tests/test-utils.js

```diff
@@ -286,6 +287,7 @@ module.exports.logCanvas2D = function logCanvas2D(enable) {
  * @param note: the data to send.  This will be converted to JSON.
  */
 module.exports.submitNote = function submitNote(key, note) {
+  note.browser = bowser;
```

Comment:
Probably it provides information about the browser to be included with notes, I see that in the commit message now.